action.yml

```diff
@@ -21,6 +21,6 @@ inputs:
     required: true
 runs:
   using: "docker"
-  image: docker://public.ecr.aws/p4v7w0a5/lazy/nodejs-action:v0.1.3
+  image: docker://public.ecr.aws/p4v7w0a5/lazy/nodejs-action:v1.0.1
```

Comment:
@vibindaniel In test actions-nodes shows that v1.0.1 image tag doesn't exist. I'm guessing it will get created after merge?

Reply:
Yes that is correct. I already modified the version because I completed the testing. You can see the test results in the pr description.